src/transaction/components/TransactionToast.tsx

```diff
@@ -36,6 +36,22 @@ export function TransactionToast({
     return 'bottom-5 left-2/4';
   }, [position]);
 
+  useEffect(() => {
+    let timer: ReturnType<typeof setTimeout> | undefined;
+    // hide toast after 5 seconds once
+    // it reaches final state (success or error)
+    if (receipt || errorMessage) {
+      timer = setTimeout(() => {
+        setIsToastVisible(false);
+      }, 5000);
```

Comment:
I feel like, this is something folks want to customize at some point. Let's create a P3 Linear ticker for this. @cpcramer this is a great ticket for you, as P3, if you don't mind.